modyn/tests/trainer_server/internal/trainer/test_pytorch_trainer.py

```diff
@@ -673,6 +669,7 @@ def test_train(
         if elapsed >= timeout:
             raise AssertionError("Did not reach desired queue state after 5 seconds.")
 
+        # TODO(@robin-oester & @MaxiBoether): this does not seem to be right...
```

Comment:
Could you elaborate here? What is the problem?

Reply:
note: there should not be identation after the waiting

Reply:
Fixed in next commit. The indentation is removed and some minor edits were done.